package.json

```diff
@@ -45,9 +44,46 @@
   },
   "type": "module",
   "exports": {
+    "./SurveyNode.svelte": {
+      "types": "./dist/SurveyNode.svelte.d.ts",
+      "svelte": "./dist/SurveyNode.svelte"
+    },
+    "./SurveyPrint.svelte": {
+      "types": "./dist/SurveyPrint.svelte.d.ts",
+      "svelte": "./dist/SurveyPrint.svelte"
+    },
+    "./SurveyQuestion.svelte": {
+      "types": "./dist/SurveyQuestion.svelte.d.ts",
+      "svelte": "./dist/SurveyQuestion.svelte"
+    },
+    "./expressions.js": {
+      "types": "./dist/expressions.d.ts",
+      "svelte": "./dist/expressions.js"
+    },
+    "./md.js": {
+      "types": "./dist/md.d.ts",
+      "svelte": "./dist/md.js"
+    },
     ".": {
       "types": "./dist/index.d.ts",
       "svelte": "./dist/index.js"
     }
-  }
+  },
+  "description": "Survey-md is an attempt to simplify creating and using surveys through the code. As a starting point, I used [surv.app project](https://surv.app) which provided the same functionality but on server side. I am trying to offer something like this out of necessity to use similar library for dynamically survey generation using markdown coming from BE. md stands for Markdown as I am using markdown language to describe the surveys. All results are stored in context object and you can easily access them using context[\"question_name\"]. There are couple of rules how to do that:",
```

Comment:
And what are the rules, where does the docu continue?

Reply:
Done